### ISSUE TITLE
Does the Skip to Content link need a title attribute?

### ISSUE THREAD

**Issue by patphg** [chrisblakley/Nebula#2063]
Current "Skip to content" link does not have a title attribute. Does it need one? It says exactly what it does in the text, but accessibility checkers protest it.

Second but slightly related, IMO it's overkill to generate the tag with a function which simply echos the HTML, could this be moved into the header.php as plain HTML? It would make it easier to customize, override, etc. in a child theme.

**Comment by patphg** [comment]
This labeled as a bug but I can't modify it, I wouldn't say this is a bug...

**Comment by chrisblakley** [comment]
That element doesn't show up in my axe reports– would you mind sharing a screenshot where it recommends the title tag? Usually title attributes aren't associated with accessibility– typically it's `alt`, `aria`, and things like `.sr-only` classes.

**Comment by chrisblakley** [comment]
Ok I moved this into the `header.php` file which resolves the core issue that it was not contained within a landmark. No need for a title or aria-label or anything because it has meaningful text within it.

For sites that have already copied header.php into the child theme, this will require a retrofit by copying and pasting the following into the `<header>` element:
```html
<a class="skip-to-content-link sr-only" href="#content-section">Skip to Content</a>
```
